### ISSUE TITLE
Quest: The Eyes of Skettis

### ISSUE THREAD

**Issue by Jobe00** [dalaranwow/dalaran-wow#5666]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The object Eye of Veil Reskk at cordinates 50, 19 cannot be looted. I loot it, no object appears in inventory, then I get the error “That is already being used.”
The other Eye of Veil Reskk in the hut at coordinates 49, 14 seems to be fine.

**Expected behaviour**: Tell us what should happen instead
I should get the quest item Eye of Veil Reskk.

**Steps to reproduce the problem**:

1. Get the quest The Eyes of Skettis.
2. Go to Veil Reskkk.
3. Attempt to loot Eye of Veil Reskk.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
The quest works. Relog and try again from a closer spot. It's a general issue with game objects, see #4998 for further info.
Closing
